libyul/backends/evm/EVMDialect.cpp

```diff
@@ -342,6 +357,36 @@ SideEffects EVMDialect::sideEffectsOfInstruction(evmasm::Instruction _instructio
 	};
 }
 
+BuiltinFunctionForEVM const* EVMDialect::verbatimFunction(size_t _arguments, size_t _returnVariables) const
+{
+	pair<size_t, size_t> key{_arguments, _returnVariables};
+	shared_ptr<BuiltinFunctionForEVM const>& function = m_verbatimFunctions[key];
+	if (!function)
+	{
+		function = make_shared<BuiltinFunctionForEVM const>(createFunction(
+			"verbatim_" + to_string(_arguments) + "_" + to_string(_returnVariables),
+			1 + _arguments,
+			_returnVariables,
+			SideEffects::worst(),
+			vector<optional<LiteralKind>>{LiteralKind::String} + vector<optional<LiteralKind>>(_arguments),
+			[=](
+				FunctionCall const& _call,
+				AbstractAssembly& _assembly,
+				BuiltinContext&,
+				std::function<void(Expression const&)> _visitExpression
+			) {
+				int diff = static_cast<int>(_returnVariables) - static_cast<int>(_arguments);
+				for (size_t i = _call.arguments.size() - 1; i > 0; --i)
+					_visitExpression(_call.arguments.at(i));
+				yulAssert(_call.arguments.size() >= 1, "");
```

Comment:
The assert should be before the loop, otherwise, we'll get into a large loop, even though `at(i)` will probably throw anyway.